src/pages/Admin/Users/Add/index.jsx

```diff
@@ -0,0 +1,338 @@
+              >
+                Username
+              </InputContainer>
+              <InputContainer
+                name="description"
+                id="description"
+                type="text"
+                onChange={handleChange}
+                value={addUserData.description}
+              >
+                Description, full name, contact, etc. (optional)
+              </InputContainer>
+              <InputContainer
+                name="email"
+                id="email"
+                type="email"
+                onChange={handleChange}
+                value={addUserData.email}
+              >
+                Email address
+              </InputContainer>
+              <InputContainer
+                type="select"
+                name="permission"
+                id="access-level"
+                onChange={handleChange}
+                options={accessLevels}
+                property="name"
+                value={addUserData.permission}
+                valueProperty="value"
+              >
+                Access level
+              </InputContainer>
+              <InputContainer
+                type="select"
+                name="folder"
+                id="root-folder"
+                onChange={handleChange}
+                options={folderlist}
+                property="name"
+                value={addUserData.folder}
+              >
+                User root folder
+              </InputContainer>
+              <InputContainer
+                name="pass1"
+                id="pass1"
+                type="password"
+                onChange={handleChange}
+                value={addUserData.pass1}
+              >
+                Password (optional)
+              </InputContainer>
+              <InputContainer
+                name="pass2"
+                id="pass2"
+                type="password"
+                onChange={handleChange}
+                value={addUserData.pass2}
+              >
+                Re-enter password
+              </InputContainer>
+
+              <p className="font-demi my-1">E-mail Notification</p>
+              <InputContainer
+                type="checkbox"
+                checked={checkboxes.enote}
+                name="enote"
+                id="enote"
+                onChange={handleChange}
+              >
+                Check to enable email notification when upload scan completes.
+              </InputContainer>
+
+              <p className="font-demi mt-1">Default upload visibility</p>
+              <CommonFields
+                accessLevel={addUserData.accessLevel}
+                handleChange={handleChange}
+              />
+
+              <label htmlFor="agents" className="font-demi my-1">
+                Agents selected by default when uploading
+                {Object.keys(checkboxes).map((key) => {
+                  if (key !== "enote")
+                    return (
+                      <InputContainer
```

Comment:
The default values for the checkboxes can be read from `agents` object of `user`

![image](https://user-images.githubusercontent.com/18077542/174746135-d92b4d2a-4b83-4e9b-b7bb-50bf3869dd6a.png)

Reply:
Got it, Will add this!

Reply:
Works perfectly :+1: